util/feeding.py

```diff
@@ -29,6 +33,14 @@ def read_csvs(csv_files):
             source_data = source_data.append(file)
     return source_data
 
+def augment(sample, sr):
+    sample = librosa.core.to_mono(np.transpose(sample, (1, 0)))
+    if random.random() < FLAGS.pitch_shift_chance:
```

Comment:
@lissyx E.g., You can see here, the if loop which go through to the modulations only happen 10% of the time each. So it's much faster than the previous iteration where it either pitch shifted or time streched every sample(50% pitch shift else other 50% time stretch. It was mutually exclusive, It is not anymore in this variant.)

Reply:
Woo, I'm not sure it's going to be okay. That seems a lot, what's your take @reuben?